srcpkgs/Orthanc/files/orthanc/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+OPTS=""
+[ -r conf ] && . ./conf
+
+: ${USER:="_orthanc"}
+
+: ${CONFIGFILE:="/etc/orthanc/Configuration.json"}
+[ -r $CONFIGFILE ] || exit 1
+
+: ${ORTHANCSTORAGE:="/var/lib/OrthancStorage"}
+if [ ! -e $ORTHANCSTORAGE ]
+then
+	mkdir -pm 0750 $ORTHANCSTORAGE &&
+	chown ${USER}:${USER} $ORTHANCSTORAGE ||
+	exit 1
+fi
+
+# Enable log
+if [ -n $LOG ]; then
+	: ${LOGDIR:="/log/orthanc"}
+	OPTS="${OPTS}--logdir=$LOGDIR "
```

Comment:
This weird trailing space behavior is likely to fail if somebody modifies `$OPTS` in `conf`. Better to do
```sh
OPTS="${OPTS} --logdir=$LOGDIR"
```
Even if `$OPTS` is empty, there's no harm in a leading space.

Reply:
I replaced to `OPTS="${OPTS} --logdir=\"$LOGDIR\""` instead.